src/js/filter.js

```diff
@@ -112,6 +112,11 @@ class Filter {
     this.onAfterUpdate()
     this.getListResults();
     setQueryParam('search', searchTerm);
+
+    gtag('event', 'search_trigger', {
+      'event_category': 'search',
+      'event_label': searchTerm
+    })
```

Comment:
How do the changes here relate to #166? Are they different approaches that do the same thing? Does this catch anything else?